<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Стеганографический Преобразователь</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/биас.ico</normaloff>../../Downloads/биас.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(184, 184, 184);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="radButMicro">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Микроточки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butChoiceHide">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 218, 218);
font: 10pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Выберите файл</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textSucc1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textInHide">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Times New Roman&quot;;
background-color: rgb(247, 247, 247);</string>
    </property>
    <property name="text">
     <string>Введите соообщение для скрытия</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radButRGB">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Алгоритм RGB</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textPathNew">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textSucc2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>190</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Times New Roman&quot;;
background-color: rgb(247, 247, 247);</string>
    </property>
    <property name="text">
     <string>Введите путь к новому файлу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butHide">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>150</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Times New Roman&quot;;
background-color: rgb(236, 236, 236);</string>
    </property>
    <property name="text">
     <string>Скрыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butChoiceFind">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 218, 218);
font: 10pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Выберите файл</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textSucc3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="butFind">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Times New Roman&quot;;
background-color: rgb(236, 236, 236);</string>
    </property>
    <property name="text">
     <string>Раскрыть</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textOutFind">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>330</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>290</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Times New Roman&quot;;
background-color: rgb(247, 247, 247);</string>
    </property>
    <property name="text">
     <string>Скрытое сообщение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonInfo">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>430</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(25, 167, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid black;</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>961</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_out">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Раскрытие информации</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>50</y>
      <width>961</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_in">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Скрытие информации</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(162, 162, 162);
font: 10pt &quot;Times New Roman&quot;;</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="for_choice">
     <property name="title">
      <string>Выбрать файл</string>
     </property>
     <addaction name="for_in"/>
     <addaction name="for_out"/>
    </widget>
    <addaction name="for_choice"/>
   </widget>
   <widget class="QMenu" name="menu_sett">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="help_list"/>
    <addaction name="for_exit"/>
    <addaction name="about_author"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_sett"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="for_in">
   <property name="text">
    <string>Для скрытия</string>
   </property>
  </action>
  <action name="for_out">
   <property name="text">
    <string>Для раскрытия</string>
   </property>
  </action>
  <action name="help_list">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="for_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="about_author">
   <property name="text">
    <string>Об авторе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
